docs/docs/recipes/integrate-logto/next-js.mdx

```diff
@@ -236,6 +236,37 @@ export const getServerSideProps = logtoClient.withLogtoSsr(
 
 <ApplyAuthorizationToken />
 
+## Edge runtime
```

Comment:
do we have links to our next sample repo?

Reply:
updated